src/Microsoft.Crank.Controller/Program.cs

```diff
@@ -36,7 +36,8 @@ public class Program
 
         private static string _tableName = "Benchmarks";
         private static string _sqlConnectionString = "";
-
+ private static string _indexName = "Benchmarks";
```

Comment:
If it's case sensitive then maybe use lowercase?

Reply:
Fixed